package.json

```diff
@@ -5,13 +5,13 @@
   "main": "lib/index.js",
   "scripts": {
     "prepublish": "not-in-publish || safe-publish-latest && npm run build",
-    "build": "rimraf lib && babel src -d lib",
+    "build": "rimraf lib && babel --source-maps inline src -d lib",
```

Comment:
https://github.com/airbnb/prop-types/blob/master/package.json#L52 is the other repo.

I'm not sure how to set it up.

It would be fine to build with inline source maps *only* when building for coverage, it just means we have to build twice.